output.txt

```diff
@@ -0,0 +1,6 @@
+You are now connected to database "nc_news" as user "nicolakirk".
```

Comment:
these shouldn't be appearing on your commits - please do not use the command 'git add . ' instead you should be only adding the files that are needed